packages/admin/cms-admin/src/pages/pageTree/MovePageMenuItem.tsx

```diff
@@ -26,13 +27,19 @@ export function MovePageMenuItem({ page }: Props): React.ReactElement | null {
         }
     `);
     const { scope } = useContentScope();
-    const { allCategories, query } = usePageTreeContext();
+    const { allCategories, query, categoryToDocumentTypesMap } = usePageTreeContext();
+    const errorDialogApi = useErrorDialog();
 
     if (allCategories.length <= 1) {
         return null;
     }
 
     const handleSubMenuItemClick = async (category: string) => {
+        if (!categorySupportsDocumentType(category, page.documentType, categoryToDocumentTypesMap)) {
+            errorDialogApi?.showError({ error: `Cannot move: Target category doesn't support documentType ${page.documentType}` });
```

Comment:
I didn't add a user message because this error is just a fallback (for disabled) and should never be shown in real life

Reply:
IMO we should throw a normal error here `throw new Error(...)`

Reply:
dbe635181be8ae4dcbe5ea5ab9e8c1498c0c7d2a